capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -8,10 +8,12 @@
  * JSON data directly from the CollegeScorecard API.
  */
 @AutoValue
-public abstract class ScorecardData {
+private abstract class ScorecardData {
 
   /**
-   * Creates a new ScorecardData
+   * Creates a new ScorecardData. The parameters come directly from the CollegeScorecard API Data
+   * Dictionary, which outlines the type and description of all available data elements. The
+   * parameters used here are a select few data elements the API provides.
```

Comment:
Can you link to 
https://collegescorecard.ed.gov/assets/CollegeScorecardDataDictionary.xlsx
in the javadoc? That way future modifiers will know where things came from.